src/api/dependency-discovery/src/github.js

```diff
@@ -0,0 +1,60 @@
+const { request } = require('@octokit/request');
+
+const hasGitHubExpired = (dateTime) => Date.now() - dateTime > 60 * 60 * 1000;
+
+const gitHubInformationRequestor = {};
+
+async function constructGitHubRequest(packageName, gitHubUrl) {
+  const labelsToSearchFor = ['hacktoberfest', 'good first issue', 'help wanted'];
+
+  const [owner, repo] = new URL(gitHubUrl).pathname.substring(1).split('/');
+
+  const requestPromises = labelsToSearchFor.map((label) => {
+    return request('GET /repos/{owner}/{repo}/issues{?assignee,state,labels}', {
+      owner,
+      repo,
+      assignee: 'none',
+      state: 'open',
+      labels: label,
```

Comment:
Does GH supports logical operators for parameters, which let you write `label1|label2`?

Reply:
Not really. That's why I have to search separately. If you pass all labels at once, GitHub is going to try and find issues that have all of those labels.

That reminds me, I think this might result in some repeated issues. I need to worry about that case, too.